pkg/virt-handler/vm_test.go

```diff
@@ -3250,12 +3252,20 @@ var _ = Describe("VirtualMachineInstance", func() {
 				}
 				allCommands = append(allCommands, cmd)
 			}
+			for _, cmdName := range OldSSHRelatedGuestAgentCommands {
+				cmd := v1.GuestAgentCommandInfo{
+					Name:    cmdName,
+					Enabled: true,
+				}
+				oldSshCommands = append(allCommands, cmd)
+			}
 			for _, cmdName := range PasswordRelatedGuestAgentCommands {
 				cmd := v1.GuestAgentCommandInfo{
 					Name:    cmdName,
 					Enabled: true,
 				}
 				allCommands = append(allCommands, cmd)
+				oldSshCommands = append(allCommands, cmd)
```

Comment:
"should succeed with SSH and only old commands" are `PasswordRelatedGuestAgentCommands` old?